webui/react/src/hooks/useModal/JupyterLab/useModalJupyterLab.tsx

```diff
@@ -122,16 +131,19 @@ const useModalJupyterLab = (): ModalHooks => {
     }
     modalClose();
     setVisible(false);
-  }, [config, fields, showFullConfig, modalClose]);
+  }, [config, form, showFullConfig, modalClose, updateDefaults]);
 
   const handleConfigChange = useCallback((config: string) => {
     setConfig(config);
     setConfigError(undefined);
   }, []);
 
-  const formContent = useMemo(() => {
-    if (!fields) return <Spinner spinning />;
+  const formContent = useMemo(
+    () => <JupyterLabForm defaults={defaults} form={form} />,
+    [defaults, form],
+  );
```

Comment:
nit: pulling this out into its own memo instead of just putting it in bodyContent makes this harder to read for seemingly no benefit

Reply:
i did it initially for what i thought was the same reason as [this](https://github.com/trentwatt/determined/blob/03ca2780dfffbf3864febeb3ddb976f588238373/webui/react/src/hooks/useModal/HyperparameterSearch/useModalHyperparameterSearch.tsx#L387) when antd was giving a warning about how the `form` wasn't attached to a `<Form />`, but I guess this wasn't the issue